hw/ip/ascon/data/ascon.hjson

```diff
@@ -646,21 +646,29 @@
     { name: "OUTPUT_VALID",
       desc: '''
         Output Valid Register
-        This register specifies which output register contains valid data and should be read next.
+        This register specifies which output register contains valid data.
         It also contains the status information whether the TAG comparison was valid or not.
       ''',
       swaccess: "ro",
       hwaccess: "hwo",
       fields: [
-        { bits: "2:0",
-          name: "DATA_TYPE",
+        { bits: "0",
+          name: "MSG_VALID",
           desc:  '''
-            Specifies which output type/register is valid.
-            There are:
-            PT_OUT, CT_OUT, TAG_OUT, NONE
+            Indicates if there is (1) or if there is not (0) valid data in the !!MSG_OUT register.
+            If !!OUTPUT_VALID.MSG_VALID and !!OUTPUT_VALID.TAG_VALID are both set,
+            !!MSG_OUT should be read before !TAG_OUT.
```

Comment:
Nit: one sentence per line.

Question: does it actually matter whether software first reads MSG_OUT or TAG_OUT in that case?

Reply:
If TAG_OUT is read before MSG_OUT everything should work, but 1) this doesn't make sense from the protocol perspective and second it reduces the potential for bugs. HW might (due to a bug) proceed with something different after the tag has been read.